hardhat.config.ts

```diff
@@ -91,12 +91,31 @@ const config: HardhatUserConfig = {
   etherscan: {
     apiKey: {
       mainnet: env.string("ETHERSCAN_API_KEY_ETH", ""),
-      goerli: env.string("ETHERSCAN_API_KEY_ETH", ""),
-      arbitrumGoerli: env.string("ETHERSCAN_API_KEY_ARB", ""),
+      sepolia: env.string("ETHERSCAN_API_KEY_ETH", ""),
+      // arbitrumSepolia: env.string("ETHERSCAN_API_KEY_ARB", ""),
```

Comment:
Maybe remove this line, if it is not used? Or worth considering adding the comment, on why this line is kept